app/models/liabilitycalculation/Calculation.scala

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.liabilitycalculation
+
+import models.liabilitycalculation.taxcalculation.TaxCalculation
+import play.api.libs.json.{Json, OFormat}
+
+case class Calculation(
+                        allowancesAndDeductions: Option[AllowancesAndDeductions],
+                        reliefs: Option[Reliefs],
+                        taxDeductedAtSource: Option[TaxDeductedAtSource],
+                        giftAid: Option[GiftAid],
+                        marriageAllowanceTransferredIn: Option[MarriageAllowanceTransferredIn],
+                        employmentAndPensionsIncome: Option[EmploymentAndPensionsIncome],
+                        employmentExpenses: Option[EmploymentExpenses],
+                        stateBenefitsIncome: Option[StateBenefitsIncome],
+                        shareSchemesIncome: Option[ShareSchemesIncome],
+                        foreignIncome: Option[ForeignIncome],
+                        chargeableEventGainsIncome: Option[ChargeableEventGainsIncome],
+                        savingsAndGainsIncome: Option[SavingsAndGainsIncome],
+                        dividendsIncome: Option[DividendsIncome],
+                        incomeSummaryTotals: Option[IncomeSummaryTotals],
+                        taxCalculation: Option[TaxCalculation]
+                      )
+
+object Calculation {
+  implicit val format: OFormat[Calculation] = Json.format[Calculation]
+}
+
+case class ChargeableEventGainsIncome(totalOfAllGains: Int)
```

Comment:
As mentioned in a previous comment, I would remove all case classes and their companion objects in their own files.

Reply:
Same as below